include/net/nrf_cloud_rest.h

```diff
@@ -312,6 +316,8 @@ int nrf_cloud_rest_jitp(const sec_tag_t nrf_cloud_sec_tag);
  * @brief Send a JSON formatted device message using the SendDeviceMessage endpoint.
  *
  * @param[in,out] rest_ctx Context for communicating with nRF Cloud's REST API.
+ *			   If a valid JWT is not provided, one will be generated.
+ *			   See @ref nrf_cloud_rest_context.auth for details.
```

Comment:
It's a toss up for me -- On the one hand, I would hate to have important information about the behavior of a function not be documented directly in that function's description

On the other hand, I do see where duplication of information leads to maintenance concerns.

In this case, since the context struct describes the context for the lib as a whole, and technically the JST autogen behavior should be universal throughout the library, you're probably right, it makes sense to remove the specific documentation on this particular function (even though this function is where that behavior is actually implemented)

I will do that!